tests/python_tests/samples/test_benchmark_vlm.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import pytest
+import sys
+
+from conftest import SAMPLES_PY_DIR, SAMPLES_CPP_DIR
+from test_utils import run_sample
+
+class TestBenchmarkVLM:
+    @pytest.mark.vlm
+    @pytest.mark.samples
+    @pytest.mark.parametrize(
+        "convert_model, download_test_content",
+        [
+            pytest.param("tiny-random-minicpmv-2_6", ("cat.png", "images")),
```

Comment:
```suggestion
            pytest.param("tiny-random-minicpmv-2_6", ("images/cat.png")),
```
That way you don't need `file_name, subdir = request.param if isinstance(request.param, tuple) else (request.param, "")`

Reply:
done

Reply:
You fixed other places but that one wasn't fixed

Reply:
yes, my bad, done